packages/twenty-server/src/engine/metadata-modules/field-metadata/services/field-metadata-related-records.service.ts

```diff
@@ -100,8 +121,108 @@ export class FieldMetadataRelatedRecordsService {
     }
   }
 
+  private computeViewFilterDisplayValue(
+    newViewFilterOptions: FieldMetadataDefaultOption[],
+  ): string {
+    if (newViewFilterOptions.length > MAX_OPTIONS_TO_DISPLAY) {
+      return `${newViewFilterOptions.length} options`;
+    }
+
+    return newViewFilterOptions.map((option) => option.label).join(', ');
+  }
+
+  public async updateRelatedViewFilters(
+    oldFieldMetadata: SelectFieldMetadataEntity,
+    newFieldMetadata: SelectFieldMetadataEntity,
+  ): Promise<void> {
+    const filters = await this.getFieldMetadataViewEntity(
+      newFieldMetadata,
+      'viewFilters',
+    );
+
+    const { updated: updatedFieldMetadata, deleted: deletedFieldMetadata } =
```

Comment:
I really think we should update the name to 
```suggestion
    const { updated: updatedOptions, deleted: deletedOptions } =
```
or 
```suggestion
    const { updated: updatedFieldMetadataOptions, deleted: deletedFieldMetadataOptions } =
```

because updatedFieldMetadata makes me expect a FieldMetadataEntity